src/app/action-items/action-items.component.css

```diff
@@ -146,6 +150,11 @@
     justify-content: center;
 }
 
+.fa-edit {
```

Comment:
**SUGGESTION:** It's usually better if we want to change the look of the generic class only in the single place to make a chain (nesting) of the css classes so eg. `.details. .edit-button .fa-edit`. Then we won't unexpectedly change the look in other places. It also explains easier why and what we want to change.

Reply:
@romatowskak what about that? still not applied